tests/flow/test_config.py

```diff
@@ -79,7 +78,21 @@ def test03_config_set(self):
         expected_response = [config_name, config_value]
         self.env.assertEqual(response, expected_response)
 
-    def test04_config_set_invalid_name(self):
+    def test04_config_set_multi(self):
```

Comment:
Please add additional tests which try to set multiple configuration fields but fail due to invalid configuration value or field.